format-registry/stream/index-respec.html

```diff
@@ -151,14 +151,14 @@
               audio/mp4
               <br/>video/mp4
             </td>
-            <td><a href="mp4.html">ISO Common Encryption ('cenc') Protection Scheme for ISO Base Media File Format Stream Format</a></td>
+            <td><a def-id="stream-registry-mp4"></a> [[!EME-STREAM-MP4]]</td>
```

Comment:
Comments for unmodified lines related to consistency with the other registry:
* Line 95: Add ` [[!ENCRYPTED-MEDIA]]`
* Line 106: Add ` [[!ENCRYPTED-MEDIA]]`
* Line 130: Add `mailto:`